src/algoliasearch.helper.js

```diff
@@ -248,6 +250,50 @@ AlgoliaSearchHelper.prototype.searchOnce = function(options, cb) {
   });
 };
 
+/**
+ * Start the search for answers with the parameters set in the state.
+ * This method returns a promise.
+ * @return {AlgoliaSearchHelper}
+ * @return {promise.<FacetSearchResult>} the answer results
+ */
+AlgoliaSearchHelper.prototype.searchForAnswers = function(attributesForPrediction, queryLanguages, nbHits) {
+  var state = this.state;
+  var derivedHelper = this.derivedHelpers[0];
+  if (!derivedHelper) {
+    return Promise.resolve([]);
+  }
+  var derivedState = derivedHelper.getModifiedState(state);
+  var data = merge(
+    {
+      attributesForPrediction: attributesForPrediction,
+      nbHits: nbHits
+    },
+    {
+      params: omit(requestBuilder._getHitsSearchParams(derivedState), [
+        'attributesToSnippet',
+        'hitsPerPage',
+        'restrictSearchableAttributes',
+        'snippetEllipsisText' // FIXME remove this line once the engine is fixed.
```

Comment:
Unlike [the doc](https://www.algolia.com/doc/rest-api/answers/#method-param-params), `snippetEllipsisText` is throwing an error. I've reported it to the team, but for now let's have it here.